gen/dcos-config.yaml

```diff
@@ -4,6 +4,7 @@ package:
       {
         "environment": {
           "MESOS_IP_DISCOVERY_COMMAND": "/opt/mesosphere/bin/detect_ip",
+          "PLATFORM": "{{ platform }}"
```

Comment:
This was purely added here... It only adds / changes these things. That test script cares about other things (Testing on AWS using IAM type things).